examples/ucnf-kiknos/scripts/start_topo.sh

```diff
@@ -3,8 +3,8 @@
 # Topology information
 CLUSTER1=${CLUSTER1:-kiknos-demo-1}
 CLUSTER2=${CLUSTER2:-kiknos-demo-2}
-VPP_AGENT=${VPP_AGENT:-ciscolabs/kiknos:latest}
-NSE_ORG=${NSE_ORG:-mmatache}
+VPP_AGENT=${VPP_AGENT:-ciscolabs/kiknos-sswan:latest}
```

Comment:
did you need to update the vppagent for any NAT feature?

Reply:
Yes, I had to update kiknos agent (https://wwwin-github.cisco.com/ccnf/kiknos/pull/37)
Also, `kiknos-sswan` is what we call kiknos v2, refactored kiknos control plane.